modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -2129,6 +2129,25 @@ public CodegenModel fromModel(String name, Schema schema) {
             return null;
         }
 
+        String modelJson = Json.pretty(schema);
+        if (schema.getType() == null ) {
+            Schema copy = ModelUtils.unaliasSchema(this.openAPI, schema, importMapping);
```

Comment:
I wonder how to clone `schema` instance here. I need to change result of `Json.pretty(schema);` but keep original instance not modified.